index.js

```diff
@@ -220,20 +220,22 @@ async function createNewExplanation(info) {
 
     // If the comic title is a number underneath the current comic number, do not create this redirect
     if (!/^\d+$/.test(comicTitle) || +comicTitle > comicNum) {
+      const comicTitleRedirect = `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`;
       await bot.edit(
         comicTitle,
-        `#REDIRECT [[${comicNum}: ${comicTitle}]]\n`,
-        EDIT_SUMMARY,
+        content,
```

Comment:
What does `content` refer to here?

Reply:
This was a messed up variable renaming. Fixed in 740f5e9.